src/Resources/contao/library/Contao/Environment.php

```diff
@@ -332,7 +292,7 @@ protected static function httpXForwardedHost()
 	 */
 	protected static function ssl()
```

Comment:
If we agree to move away from the current `Environment` implementation, I will be eager to change this.
For the moment it is, as you stated, a 1:1 translation.